db/migrate_cards/20161005120800_metric_answer_lookup_table.rb

```diff
@@ -12,13 +12,14 @@ def up
       t.integer  :year
       t.string   :metric_name
       t.string   :company_name
-      t.string :designer_name
-      t.string :title_name
+      t.string   :designer_name
+      t.string   :title_name
       t.string   :metric_record_name
       t.string   :value
+      t.float    :numeric_value
       t.datetime :updated_at
       t.boolean  :imported
-      t.boolean :latest
+      t.boolean  :latest
     end
 
     add_index :metric_answers, :metric_answer_id,
```

Comment:
I know it's a bit late to say this, and we probably don't want to do anything right away, but it's been occurring to me that perhaps we should use "Record" "Answer" and "Answer Value" in code.  Just flagging this to discuss before we set about any grand renaming.